src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp

```diff
@@ -406,15 +450,15 @@ void compare(const ov::Tensor& expected,
         }
 
         double abs = std::fabs(expected_value - actual_value);
-        double rel = expected_value ? (abs / std::fabs(expected_value)) : abs;
-
+        double rel =
+            expected_value && actual_value && !std::isinf(expected_value) ? (abs / std::fabs(expected_value)) : 0;
         abs_error.update(abs, i);
         rel_error.update(rel, i);
     }
     abs_error.mean /= shape_size_cnt;
     rel_error.mean /= shape_size_cnt;
 
-    if (!(less_or_equal(abs_error.max, abs_threshold) && less_or_equal(rel_error.max, rel_threshold))) {
+    if (!(less_or_equal(abs_error.max, abs_threshold) || less_or_equal(rel_error.mean, rel_threshold))) {
```

Comment:
@iefode This PR leads to some failures of plugin unit test pass. Would you please check again?

Reply:
@ceciliapeng2011 In your case, please update your branch to latest master. Commit was reverted